pkg/compactor/blocks_cleaner.go

```diff
@@ -27,6 +27,7 @@ import (
 
 const (
 	defaultDeleteBlocksConcurrency = 16
+	reasonLabelValue               = "retention"
```

Comment:
Just curious, the possible values for this are `retention` and `compaction`? What's the main usecase to know whether a block is marked for deletion due to retention or compaction?

And a nit about the naming. Should we call it `reasonValueRetention` instead?

Reply:
It was there before this change. In my PR, I tried maintaining the original label values. My guess of the purpose of `compaction` label value is that end user tries to know how many blocks were deleted after compaction (at the end of Thanos compact code, source blocks would be marked for deletion if compaction succeeded). The purpose of `retention` label value is the number of blocks marked for deletion during cleaner runtime.